Kitodo/src/main/webapp/WEB-INF/templates/includes/processes/progressColumn.xhtml

```diff
@@ -19,7 +19,7 @@
     <h:panelGroup
             class="process-progress"
             title="#{ProcessForm.getCurrentTaskTitles(process)}"
-            rendered="#{not empty process.tasks}"
+            rendered="#{not empty process.tasks or process.hasChildren()}"
```

Comment:
What happens, if the process has children, but they don't have tasks? (e.g year processes)

Reply:
This does not seem to work as intended, see #5171